compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -462,5 +587,11 @@ def cli_run():
     app()
 
 
+def cli_run_funcx():
+    """Entry point that prints a custom message. i.e. deprecation warnings"""
+    print(DEPRECATION_FUNCX_ENDPOINT)
+    app()
+
+
```

Comment:
Where is `cli_run_funcx()` utilized?

Reply:
Question retracted&nbsp;&mdash;&nbsp;explained verbally.  Thanks!

My one suggestion is to perhaps think about drawing attention to this on the command line.  Otherwise, I can see folks ignoring the "drab gray text" because the output looks basically the same as that to which they're already accustomed.  Perhaps something super obnoxious like:

```python
def cli_run_funcx():
    fmt = DEPRECATION_FUNCX_ENDPOINT
    fmt = "{title}DEPRECATION NOTICE{frs}\n{body}" + fmt + "{rs}"
    title = frs = body = rs = ""
    if sys.stderr.isatty():
        title = "\033[91;40m"
        body = "\033[93;40m"
        frs = "\033[39m"
        rs = "\033[0m"

    print(fmt.format(title=title, body=body, frs=frs, rs=rs), file=sys.stderr)
    app()
```

Reply:
Added but changed colors a bit to show up better if light color scheme.  
![Screenshot 2023-04-07 at 11 23 34 AM](https://user-images.githubusercontent.com/101125281/230635280-cf30ab09-1c25-4ea2-94dd-8502b73c207b.png)

Reply:
The top is bright yellow which is great on dark, but almost unreadable on light.  Deprecation notice with white/red shows well in both compared to red/black.

Reply:
No problem on the color choices.  (FWIW, that was why I had also set the background color in my suggestion.)

Looks good!